src/test/unit/services/sample/standalone_gqs_test.cpp

```diff
@@ -2,159 +2,126 @@
 #include <stan/services/sample/standalone_gqs.hpp>
 #include <stan/callbacks/stream_writer.hpp>
 #include <stan/callbacks/stream_logger.hpp>
-#include <stan/io/empty_var_context.hpp>
-#include <test/test-models/good/services/test_gq.hpp>
+#include <stan/io/dump.hpp>
+#include <stan/io/stan_csv_reader.hpp>
+#include <test/test-models/good/services/bernoulli.hpp>
 #include <test/unit/services/instrumented_callbacks.hpp>
 #include <test/unit/util.hpp>
+#include <boost/algorithm/string.hpp>
+#include <Eigen/Dense>
 #include <gtest/gtest.h>
 #include <iostream>
+#include <vector>
 
-class ServicesStandaloneGQ : public testing::Test {
+class ServicesStandaloneGQ : public ::testing::Test {
 public:
-  ServicesStandaloneGQ()
-    : model(context, &model_log) {}
+  ServicesStandaloneGQ() :
+    logger(logger_ss, logger_ss, logger_ss, logger_ss, logger_ss) {}
+
+ void SetUp() {
+  std::fstream data_stream("src/test/test-models/good/services/bernoulli.data.R",
+                           std::fstream::in);
+  stan::io::dump data_var_context(data_stream);
+  data_stream.close();
+  model = new stan_model(data_var_context);
```

Comment:
PS Bravo figuring out how to run actual compiled model code and do gtest test fixtures 👍